package.json

```diff
@@ -4,16 +4,19 @@
   "private": true,
   "author": "Matthew Brandly",
   "license": "MIT",
+  "main": "src/angular-youtube-embed.js",
```

Comment:
I disagree, but feel free to do it how you like. Since I use packing tools with CommonJS stuff (webpack, browserify, etc.) I always prefer unminified files as `main` so it's easier to debug when it gets inlined (the sourcemap makes sense) and it keeps things neater, in my opinion.